files/helpers.py

```diff
@@ -501,10 +503,9 @@ def get_base_ffmpeg_command(
         enc_type {str} -- encoding type (twopass or crf)
     """
 
-    target_fps = int(target_fps)
-    # avoid Frame rate very high for a muxer not efficiently supporting it.
-    if target_fps > 90:
-        target_fps = 90
+    # avoid very high frame rates
+    while target_fps > 60:
+        target_fps = target_fps/2
```

Comment:
What's the rationale for halving it? I guess it makes the pulldown easier.

Reply:
Yes it is easier and you actually do not have to do a pull down of any sort, this just makes the encoded video to show every other, fourth, eight...frame, so no blending of frames is necessary. I set it to 60 as this is nowadays the maximum what displays can in most cases render and it is requires less CPU power to encode/decode. We can raise it to 120 in future when 120Hz displays become the mainstream.